app/src/test/java/com/example/android/architecture/blueprints/todoapp/statistics/StatisticsUtilsTest.kt

```diff
@@ -83,7 +81,6 @@ class StatisticsUtilsTest {
     }
 
     @Test
-    @Ignore
```

Comment:
Seems a bit out of scope of the change?

Reply:
True, I've moved it to a different commit so it's cherry-pickable but still part of this PR.